src/components/Modals/components/Modal.js

```diff
@@ -62,7 +65,8 @@ export const Modal = ({
                 {cancelButtonLabel}
               </button>
               <button
-                type="button"
+                type={okButtonType || "button"}
```

Comment:
nice find. does submit not work for the other browsers?

Reply:
`submit` is well supported, but it doesn't trigger the onSubmit callback and give you form validation like `requestSubmit` does.

Reply:
does make sense. thanks for the explanation.

i was just not sure if we somehow can avoid passing the `okButtonType`.